package.json

```diff
@@ -16,13 +15,13 @@
     "bootstrap-evergreen": "npm ci && lerna run prepare",
     "start": "npm run start --workspace=mongodb-compass",
     "install-electron": "cross-env ELECTRON_SKIP_BINARY_DOWNLOAD= lerna exec --concurrency 1 -- node ../../scripts/install-electron.js",
-    "amend-compass-version": "node scripts/bump-compass-dependencies.js && git commit --no-allow-empty -m \"chore(release): sync compass dependencies\" packages/compass/package.json",
-    "packages-version": "lerna version --no-private --no-push -m \"chore(release): bump package versions\"",
-    "postpackages-version": "npm run amend-compass-version && git push && git push --tags",
-    "packages-publish": "node scripts/monorepo/publish-all.js",
-    "packages-test-version": "lerna version --no-private --allow-branch=* --no-git-tag-version --no-push",
-    "postpackages-test-version": "node scripts/bump-compass-dependencies.js",
-    "packages-test-publish": "node scripts/monorepo/publish-all.js --local --no-pristine-check --no-branch-check",
+    "packages-version": "lerna version --no-push --no-private -m \"chore(release): Bump package versions\"",
+    "test-packages-version": "npm run packages-version --",
+    "version": "npm install --package-lock-only && git add package-lock.json",
```

Comment:
Will run before lerna tags and commits version updates, but after they were updated already, so perfect place to sync package-lock file